custom-scrollbar.html

```diff
@@ -71,13 +75,29 @@
         cursor: default;
         outline: none;
         z-index: 1;
-        opacity: 0.45;
+        opacity: 0.2;
         transform: translateY(0);
         background: black;
-        transition: height 0.2s;
```

Comment:
Can you rebase your commit without update default value of opacity and transition ?

Reply:
k sure